packages/block-editor/src/components/preview-options/README.md

```diff
@@ -82,6 +82,13 @@ Used to set the device type that will be used to display the preview inside the
 -   Type: `func`
 -   Required: yes
 
+#### children
+
+A function that returns nodes to be rendered within the dropdown.
+
+-   Type: `Function`
+-   Required: No
+
```

Comment:
This definition was inspired by [the AutoComplete component](https://github.com/WordPress/gutenberg/blob/b606951aac8e513bd48022b04c0c002978fd87ad/packages/components/src/autocomplete/README.md?plain=1#L44-L49).